### ISSUE TITLE
Fetch SQL in SQL Insert step can produce error

### ISSUE THREAD

**Issue by slawo-ch** [twineworks/tweakstreet-issues#4]
### Description

When pressing the "Fetch SQL" button on the SQL Insert step - to get the DDL for table creation - an error is thrown.

### Steps to reproduce

 - create a new SQL Insert step
 - open its properties
 - press fetch SQL

### Expected behavior

The DDL for creating the target table should be put into the SQL editor.

### Actual behavior

The following error is thrown instead:

```
could not find target name settings.pkField in runtime module
INFO: 
	at: unknown
	code: RUNTIME_ERROR
	message: could not find target name settings.pkField in runtime module
	value: nil
ERROR: 
	code: java.lang.NullPointerException
	message: could not find target name settings.pkField in runtime module
``` 

### Environment

* Tweakstreet Version: 1.4.0

**Comment by slawo-ch** [comment]
resolved as part of v1.4.1

**Comment by uwegeercken** [comment]
works! thank you.
